### PR TITLE
DOC: add collection-fontsrecommended to dependencies

### PR DESCRIPTION
**Pull request by story645**
My latex and fonts have imploded and started deleting themselves and doing a minimal texlive install, I also needed ``collection-fontsrecommended` for the docs to not error out.

### CODE REVIEW

**Comment by tacaswell** [comment]
Could you tell from the error messages which font was missing?  The packaging of latex stuff seems to vary significantly between ecosystem, could you also add a note which distro this naming is from?

File: doc/devel/dependencies.rst [hunk 1]
```diff
@@ -414,6 +414,9 @@ Required
   * `dvipng <https://ctan.org/pkg/dvipng>`_
   * `underscore <https://ctan.org/pkg/underscore>`_
 
+* if using a minimal install of TeX-Live, you may also need ``collection-fontsrecommended``
```
Comment (jklymak) [inline doc/devel/dependencies.rst:417]: I would put this under the TeX-Live bullet.  "A LaTeX distribution that includes at least a, b, c, e.g., ..."

**Comment by QuLogic** [comment]
You can find an exact list at https://src.fedoraproject.org/rpms/python-matplotlib/blob/rawhide/f/python-matplotlib.spec#_95 though that's probably far too specific to list for users. But you might be able to convert those into the collections.

File: doc/users/installing/dependencies.rst [hunk 2]
```diff
@@ -397,28 +397,32 @@ External tools
...
 * `Graphviz <http://www.graphviz.org/download>`_
-* a minimal working LaTeX distribution, e.g., `TeX Live <https://www.tug.org/texlive/>`_ or
+* a minimal working LaTeX distribution, e.g. `TeX Live <https://www.tug.org/texlive/>`_ or
   `MikTeX <https://miktex.org/>`_
-* the following LaTeX packages (if your OS bundles TeX Live, the
-  "complete" version of the installer, e.g. "texlive-full" or "texlive-all",
-  will often automatically include these packages):
 
-  * `cm-super <https://ctan.org/pkg/cm-super>`_
+The following LaTex packages:
```
Comment (QuLogic) [inline doc/users/installing/dependencies.rst:409]:
```suggestion
The following LaTeX packages:
```

**Review by QuLogic — APPROVED** [review]

**Pull request merged**